HesoyamHospital/WebApplication/HospitalSurvey/SurveyController.cs

```diff
@@ -89,19 +135,19 @@ public IActionResult FrequenciesPerQuestion(string section)
         {
             if (section == "Doctor")
```

Comment:
Perhaps we could replace multiple if/else-if with switch?
Using IFs is better when we work with booleans.